scopes/code/ui/code-compare/code-compare-view/code-compare-view.module.scss

```diff
@@ -0,0 +1,48 @@
+.diffEditor {
+  // TODO - remove unneeded css vars overrides
+  // TODO - find a better place to put the color overrides. maybe move to a theme?
```

Comment:
ughhhhhhh

Reply:
try replacing these with one of these:
https://bit.cloud/teambit/base-ui/theme/dark-theme
https://bit.cloud/teambit/design/themes/dark-theme